includes/admin/class-wc-rest-payments-webhook-controller.php

```diff
@@ -269,11 +269,27 @@ private function process_webhook_expired_authorization( $event_body ) {
 	 * @throws Invalid_Payment_Method_Exception When unable to resolve charge ID to order.
 	 */
 	private function process_webhook_payment_intent_failed( $event_body ) {
+		// Check to make sure we should process this according to the payment method.
+		$payment_method     = $this->get_payment_method_from_event( $event_body );
+		$actionable_methods = [
+			'us_bank_account',
+		];
```

Comment:
What do you think about using `WCPay\Constants\Payment_Method` rather than hardcoded strings here?

Reply:
Do you mean add more methods to the `Payment_Method` constants? If so, I think we can try that.

Reply:
I already added BECS to it. But I mean to use the constant there rather than the id as a string.

Reply:
Yup, that's what I meant, I just didn't say it clearly. Updated in https://github.com/Automattic/woocommerce-payments/pull/3650/commits/c9e696bd53ac8311f3d974e2e5baf9c564e36dbf